index.html

```diff
@@ -300,6 +300,25 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
             </div>
           </div>
         </div>
+      </div>
+       <div class="col-md-4">
+        <div class="profile-section">
+          <div class="text-center">
+            <img src="./assets/images/hamdaan.jpg" alt="Sandeep Velayudhan" class="profile-image">
```

Comment:
@githamdaan25 
Is the `alt` attribute value is correct?

Reply:
corrected it